packages/grid-pro/theme/lumo/vaadin-grid-pro-styles.js

```diff
@@ -46,6 +46,27 @@ registerStyles(
         var(--lumo-contrast-5pct) 14px
       );
     }
+
+    /* Loading editor cell styles are used by Flow GridPro */
+    :host([loading-editor]) [part~='focused-cell']::before {
+      content: '';
+      position: absolute;
+      inset: 0;
+      pointer-events: none;
+      box-shadow: inset 0 0 0 var(--_focus-ring-width) transparent;
+      animation: vaadin-grid-pro-loading-editor 1.4s infinite;
+    }
+
+    :host([loading-editor]) [part~='focused-cell'] ::slotted(vaadin-grid-cell-content) {
+      opacity: 0;
+      pointer-events: none;
+    }
+
+    @keyframes vaadin-grid-pro-loading-editor {
+      50% {
+        box-shadow: inset 0 0 0 var(--_focus-ring-width) var(--_focus-ring-color);
+      }
+    }
```

Comment:
Are we going to add styles for the Material theme as well?

I wonder if the part that sets the `opacity` and `pointer-events` should be moved to the core styles of the component.

Reply:
I think at least pointer-events could be in the core styles since this is covered by unit test.

Reply:
> I wonder if the part that sets the opacity and pointer-events should be moved to the core styles of the component.

Probably still makes sense to move opacity and pointer-events to core styles. Done.